docs/zh-cn/auth.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Authorization
+keywords: Authorization
+description: Authorization
+---
+
+# 鉴权
+
+## 服务端如何开启鉴权
+
+### 非Docker环境
+
+按照官方文档配置启动,默认是不需要登录的，这样会导致配置中心对外直接暴露。而启用鉴权之后，需要在使用用户名和密码登录之后，才能正常使用nacos。
+
+开启鉴权之前，application.properties中的配置信息为：
+```java
+### If turn on auth system:
+nacos.core.auth.enabled=false
+```
+开启鉴权之后，application.properties中的配置信息为：
+```java
+### If turn on auth system:
+nacos.core.auth.enabled=true
+```
+
+
+### Docker环境
+

```

Comment:
In Docker, nacos can open auth by setting `NACOS_AUTH_ENABLE` env variable.

you can add this way like:

```
#### Official images

If you use nacos official images. You can open auth with set ENV `NACOS_AUTH_ENABLE=true`.

#### Custom images

....
```